travis-after-success.sh

```diff
@@ -27,7 +27,7 @@ elif [ "$TRAVIS_EVENT_TYPE" == "cron" ]; then
   echo "Skipping snapshot deployment; running cron job."
 else
   echo "Deploying snapshot..."
-  ./gradlew clean uploadArchives
+  ./gradlew clean publishAllPublicationsToMavenRepository
```

Comment:
Should this be `publishRemotePublicationToMavenRepository`?

Reply:
Those are equivalent: we explicitly skip the `local` publication for remote repository uploads, so `All` here is effectively just `Remote`.

(Conversely, we also skip the `remote` publication for local repository installs.)